### ISSUE TITLE
Unusual generation of the HOTKEYF_EXT constant on Windows

### ISSUE THREAD

**Issue by fgimian** [microsoft/CsWin32#963]
## Actual behavior

I'm compiling on Windows and the generated constant is:

```c#
internal const uint HOTKEYF_EXT = 128U;
```

## Expected behavior

According to the metadata at https://github.com/microsoft/win32metadata/blob/81344326b62096e7c7d62bc9905d4258103eef1d/generation/WinSDK/RecompiledIdlHeaders/um/CommCtrl.h#L2932 and [the relevant Microsoft documentation](https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-sethotkey#parameters), this should be `0x08` on Windows systems.

## Repro steps

1. `NativeMethods.txt` content:

```
IShellLinkW
IPersistFile
ShellLink
MAX_PATH
SLGP_FLAGS
HOTKEYF_CONTROL
HOTKEYF_ALT
HOTKEYF_SHIFT
HOTKEYF_EXT
VIRTUAL_KEY
```

2. `NativeMethods.json` content (if present):

N/A

3. Any of your own code that should be shared?

N/A

### Context

- CsWin32 version: 0.3.2-beta
- Win32Metadata version (if explicitly set by project): n/A
- Target Framework: net7.0-windows
- `LangVersion` (if explicitly set by project): N/A

Thanks heaps in advance!
Fotis

**Comment by elachlan** [comment]
This is how its defined:
```
#ifdef _MAC
#define HOTKEYF_EXT             0x80
#else
#define HOTKEYF_EXT             0x08
#endif
```
It looks like its picked up the first value. I have no idea what `_MAC` means.

@mikebattista might know whats going on.

**Comment by fgimian** [comment]
> This is how its defined:
> 
> ```
> #ifdef _MAC
> #define HOTKEYF_EXT             0x80
> #else
> #define HOTKEYF_EXT             0x08
> #endif
> ```
> 
> It looks like its picked up the first value. I have no idea what `_MAC` means.
> 
> @mikebattista might know whats going on.

Yeah I noticed that too and thought it was unusual. Now that I look, it seems that Rust for Windows is also [generating it incorrectly](https://microsoft.github.io/windows-docs-rs/doc/windows/Win32/UI/Controls/constant.HOTKEYF_EXT.html) so I wonder if it's related to the metadata itself.

**Comment by elachlan** [comment]
I'd say it is a metadata issue with the scraper. Either @aarnott or @mikebattista  will know more and should be able to move this issue to win32metadata.

**Comment by AArnott** [comment]
Yes, this is an issue in the metadata -- not the projection. I'll transfer the issue.
